src/clr-angular/data/datagrid/datagrid.html

```diff
@@ -53,7 +53,7 @@
 </div>
 <ng-content select="clr-dg-footer"></ng-content>
 <div class="datagrid-spinner" *ngIf="loading">
-    <div class="spinner spinner-md">Loading...</div>
+    <clr-spinner medium>Loading...</clr-spinner>
```

Comment:
I think we don't want to have `...` as it gets read out `period period period` or something. Should probably use ellipsis instead `&hellip;`